### ISSUE TITLE
Modify the cropX cropY parameter that contains the clipPath object, it not be updated after canvas.renderAll()

### ISSUE THREAD

**Issue by xh4010** [fabricjs/fabric.js#6907]
ClipPath is applied to my picture, and the cropX cropY parameter needs to be dynamically modified, but when I modify the parameter, it not be updated after canvas.renderAll().

## Version
4.3.1

## Test Case
https://codepen.io/xh4010/pen/ZEBjrqe

## Information about environment
browser

**Comment by asturur** [comment]
i see the cache does not get invalidated.
It could be that for the image, cropX and cropY aren't in the cache properties.

**Comment by xh4010** [comment]
Hi, @asturur
thank you for your reply.
After indeed modifying cropX and cropY, the modification is not marked, and dirty is still false. My current solution is to set dirty=true together:
obj.set({
  cropX: 80,
  cropY: 120,
  dirty: true
})
see:  https://codepen.io/xh4010/pen/ZEBjrqe
Thanks

**Comment by asturur** [comment]
yes cropX and cropY should be in the cacheProperties array in the image class. This could be easily fixed if someone could open a pr for it.

@stefanhayden @melchiar @rileygowan 

i pinged 3 brave friendly guys that maybe can do it.
